lib/routes/gxmzu/utils/index.ts

```diff
@@ -26,15 +26,27 @@ async function getNoticeList(ctx, url, host, titleSelector, dateSelector, conten
                     item.description = '该通知无法直接预览，请点击原文链接↑查看';
                 } else {
                     const response = await got({ url: item.link, https: { rejectUnauthorized: false } });
-                    if (response.redirectUrls.length) {
+                    if (response.redirectUrls && response.redirectUrls.length > 0) {
```

Comment:
`res.redirectUrls` will always be `undefined` since #14922. `'@/utils/got'` is now a partial-got-like wrapper of [ofetch](https://github.com/unjs/ofetch).

Reply:
> `res.redirectUrls` will always be `undefined` since #14922. `'@/utils/got'` is now a partial-got-like wrapper of [ofetch](https://github.com/unjs/ofetch).

Thank you for your note on the updates.

I have updated the code by replacing the `got` library with the `ofetch` library. In light of the change in the handling of redirects since the update #14922, where `res.redirectUrls` will always be undefined, I've adapted our redirect handling logic to now work based on HTTP status codes instead. This change ensures that our approach aligns with the current functionality provided by the `ofetch` library, effectively managing redirects without relying on the `redirectUrls` property.

Let me know if there are any further adjustments or reviews needed. Thank you for your attention to this matter!